src/reporter.h

```diff
@@ -48,6 +48,11 @@ class MetadataReporter {
   void SendMetadata(std::vector<MetadataStore::Metadata>&& metadata)
       throw (boost::system::system_error);
 
+  // Create the fully qualified location based on the raw location reported by
+  // the metadata server.
+  const std::string FullyQualifiedResourceLocation(const std::string& location)
```

Comment:
The first `const` is superfluous and can go away. Then this fits on one line.

Reply:
done